assets/js/theme/global/currency-selector.js

```diff
@@ -1,5 +1,57 @@
-export default function () {
+import swal from './sweet-alert';
+import utils from '@bigcommerce/stencil-utils';
+
+export default function (cartId) {
+    function changeCurrency(url, currencyCode) {
+        $.ajax({
+            url,
+            method: 'POST',
+            data: JSON.stringify({ currencyCode }),
+        }).then(() => {
+            window.location.reload();
+        });
+    }
+
     $(document.body).on('click', '.currencySelector', () => {
         $('.currency-selection-list').toggleClass('active');
     });
+
+    $(document.body).on('click', '.dropdown-menu-item a', event => {
```

Comment:
I think it'd be better if you use a more specific selector rather than targeting a generic CSS selector. In this case, you can just target `[data-cart-currency-switch-url]`.